tokenserver/views.py

```diff
@@ -96,6 +96,13 @@ def valid_assertion(request):
             raise _unauthorized("invalid-timestamp")
         raise _unauthorized("invalid-credentials")
 
+    try:
+        tokenVerified = assertion['idpClaims']['fxa-tokenVerified']
+    except KeyError:
+        tokenVerified = True
```

Comment:
It would also be worth adding a comment to explain what's going on here, and why we default to `True`.